README.rst

```diff
@@ -83,22 +83,24 @@ Build and Install
 Requirements
 ------------
 
-- Operating systems: CentOS (RedHat) and Ubuntu (Debian), being CentOS 6.3 the
-  reference operating system.
+Operating systems:
 
-To install this module you have to install some components:
+- CentOS (RedHat) and Ubuntu (Debian), being CentOS 6.3 the reference
+  operating system.
+
+Software dependencies:
 
 - Python 2.7
 - PyClips 1.0 (http://sourceforge.net/projects/pyclips/files/)
 - RabbitMQ Server 3.3.0 or above (http://www.rabbitmq.com/download.html)
-- pip installed (http://docs.python-guide.org/en/latest/starting/install/linux/)
 - MySQL 5.6.14 or above (http://dev.mysql.com/downloads/mysql/)
+- pip 7.0.0 or above (https://pip.pypa.io/en/stable/installing/)
 - gcc-c++ and gcc libraries
 
 This module also needs the installation of these other components:
 
 - Fiware-facts module installed (https://github.com/telefonicaid/fiware-facts)
-- A running instance of Orion Context Broker v0.28
+- A running instance of Orion Context Broker v0.28 or above
```

Comment:
No no, above is not allowed due they do not support the functionality that we need.